.eslintrc.yml

```diff
@@ -1,5 +1,5 @@
 ---
-extends: "@elastic/kibana"
+extends: ['@elastic/eslint-config-kibana', 'plugin:@elastic/eui/recommended']
```

Comment:
This is extending the same lint settings as used by Dashboards` lint file specified [here](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/.eslintrc.js#L112)